src/components/App/App.jsx

```diff
@@ -1,80 +1,89 @@
-import './App.css'
-import React from 'react'
-import Banner from '../Banner/Banner'
-import Movies from '../Movies/Movies'
-import MovieDetails from '../MovieDetails/MovieDetails'
-import Error from '../Error/Error'
+import "./App.css";
+import React from "react";
+import Banner from "../Banner/Banner";
+import Movies from "../Movies/Movies";
+import MovieDetails from "../MovieDetails/MovieDetails";
+import Error from "../Error/Error";
+import { Route } from "react-router-dom";
 
 class App extends React.Component {
   constructor() {
-    super()
+    super();
     this.state = {
       movies: [],
       currentMovie: {},
       loading: true,
       hasError: false,
-    }
+    };
   }
 
   async componentDidMount() {
     try {
       let response = await fetch(
-        'https://rancid-tomatillos.herokuapp.com/api/v2/movies'
-      )
+        "https://rancid-tomatillos.herokuapp.com/api/v2/movies"
+      );
       if (!response.ok) {
-        throw new Error('weird error')
+        throw new Error("weird error");
       }
-      let data = await response.json()
-      this.setState({ movies: data.movies })
-      this.setState({ loading: false })
-    } 
-    catch (e) {
-      this.setState({ loading: false })
-      this.setState({ hasError: true })
+      let data = await response.json();
+      this.setState({ movies: data.movies });
+      this.setState({ loading: false });
+    } catch (e) {
+      this.setState({ loading: false });
+      this.setState({ hasError: true });
     }
   }
 
   handleClick = (event) => {
+    console.log(event);
     const movieFound = this.state.movies.find(
-      (item) => `${item.id}` === event.target.id
-    )
-    this.setState({ currentMovie: movieFound })
-  }
-
-  closeModal = () => {
-    this.setState({ currentMovie: {} })
-  }
+      (item) =>
+        `${item.id}` === event.target.id ||
+        `${item.id}` === event.target.firstChild.id
+    );
+    this.setState({ currentMovie: movieFound });
+  };
```

Comment:
Currently, this is throwing an error for me on `line 42`. I found it to be because the logical operator checks before conditionals and since `firstChild` doesn't always exist, when it tries to look for `id of firstChild` it errors because it's looking for `id of undefined`.

This can be fixed by adding optional chaining -> 
`${item.id}` === event.target.firstChild?.id

I'm about to make a PR after this one is merged, so I will add it in there.